db/model/subject.js

```diff
@@ -187,7 +187,7 @@ module.exports = function subject(seq, dataTypes) {
         }));
         Subject.addScope('absolutePath', (value) => ({
           where: {
-            absolutePath: value,
+            absolutePath: { $iLike: value },
```

Comment:
I updated the scope to be case insensitive. Any objection to having this scope be case-insensitive for all queries?

Reply:
no objection from me--doesn't break any existing tests, right?

Reply:
right.